core/src/checks/wycheproof_checks.c

```diff
@@ -0,0 +1,174 @@
+#include "config.h"
+#include "checks.h"
+#include "ecdsa.h"
+#include "log.h"
+#include "memzero.h"
+#include "secp256k1.h"
+
+#include <stdbool.h>
+
+// This function was copied from the trezor-firmware library, and was not included
+// in the initial fork. 
+// https://github.com/trezor/trezor-firmware/blob/fee0d70211e89629e682f6cda95a83d17ea88825/crypto/ecdsa.c#L1206
+//
+// ** NOTE **: ONLY USE THIS FUNCTION FOR TESTING.
+// 
+// Parse a DER-encoded signature. We don't check whether the encoded integers
+// satisfy DER requirements regarding leading zeros.
+static int ecdsa_sig_from_der(const uint8_t *der, size_t der_len, uint8_t sig[64]) {
+  memzero(sig, 64);
```

Comment:
nit: is there a named constant for this `64` somewhere? If not, should we add one?

Reply:
i'll do it as a follow up PR